applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -293,6 +310,27 @@ int cloud_codec_init(struct cloud_data_cfg *cfg, cloud_codec_evt_handler_t event
 int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
 				      struct cloud_data_neighbor_cells *neighbor_cells);
 
+#if defined(CONFIG_LOCATION_METHOD_WIFI)
+/**
+ * @brief Encode cloud codec Wi-Fi access points data.
+ *
+ * @note LwM2M builds: This function does not output a list of objects, unlike other
```

Comment:
Since there is no support for lwm2m yet this comment doesn't apply.

Reply:
I can remove this but do we remember to add it back when we have WiFi support for LwM2M?

Reply:
The comment depends on if we need to reference the paths in the corresponding lwm2m `send` API. If this is not necessary this comment applies. I'm not sure about the status quo of lwm2m wifi support.

Reply:
I just assumed it'll work similarly to neighbor cells, A-GPS and P-GPS.
But I can remove this if we are unsure.

Reply:
Yes, its likely. I did a quick look a the lwm2m APIs. Lets take the chance 🚀